app/controllers/api/v2/app_groups_controller.rb

```diff
@@ -59,6 +59,78 @@ def cluster_templates
     render json: cluster_templates
   end
 
+  def profile_app
+    profiles = []
+    AppGroup.active.all.each do |appGroup|
+      environment = appGroup.environment
+      helm_infra = appGroup.helm_infrastructure
+
+      infra_values = helm_infra.values.to_json
+      object = JSON.parse(infra_values, object_class: OpenStruct)
+
+      replication_factor = environment == "production" ? 2 : 1
```

Comment:
I don't think we can use this for the existing one. not all existing app groups have this value. can you verify this? you can either check from `helm_infra.values.kafka.count` or from the helm_template, if the template na,e contains staging or production

Reply:
Mas, it's little bit confusing to get the environment value because not all the helm_cluster_template have the Kafka count defined in their manifests. Moreover, cluster templates do not have an environment defined in their manifest. One way to find the environment is from the name of the cluster templates defined like (Production - 5.25xLarge - 2560 - 1536).

Reply:
can we use `if helm_infra_template.include? "production"` ?

so if we changes the position format of our template name, it wont break this code.

Reply:
done mas